### PR TITLE
Add dynamic JS includes

### PR DESCRIPTION
**Pull request by kitsuta**
This is required to include javascript needed to detect barcode scans.

### CODE REVIEW

File: uber/templates/base.html [hunk 1]
```diff
@@ -21,6 +21,11 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     {% block sectionStyle %}{% endblock %}
     {% block additional_styles %}{% endblock %}
+    {% if 'preregistration' not in c.PAGE_PATH %}
```
Comment (kitsuta) [inline uber/templates/base.html]: Unfortunately there's no way to do this inside the includes themselves, which is what I would've preferred.

File: uber/config.py [hunk 2]
```diff
@@ -562,3 +562,5 @@ def _is_intstr(s):
     c.VOLUNTEER_CHECKLIST = [url for step, url in _items]
 
 stripe.api_key = c.STRIPE_SECRET_KEY
+
+c.JAVASCRIPT_INCLUDES = []
```
Comment (EliAndrewC) [inline uber/config.py]: I recommend adding a comment here, something like 

```python
# plugins can append paths here which will be included as <script> tags, e.g. if a plugin
# appends '../static/foo.js' to this list, that adds <script src="../static/foo.js"></script> to
# all of the pages on the site except for preregistration pages (for performance)
```

**Comment by EliAndrewC** [comment]
+1 except for a suggested comment to be added, which you can copy/paste from my suggestion and/or write your own

**Comment by kitsuta** [comment]
Oh, good point! I added a slightly reworded version of your suggestion.

**Comment by EliAndrewC** [comment]
+1

**Pull request merged**